examples/qt_types_standalone/src/main.cpp

```diff
@@ -287,14 +322,14 @@ TEST_CASE("Can copy a QSizeF on the Rust side")
 {
   const auto s = QSizeF(1.23, 4.56);
   const auto c = copy_qsizef(s);
-  CHECK(c.width() == 1.23);
-  CHECK(c.height() == 4.56);
+  CHECK(qFuzzyCompare(c.width(), 1.23));
+  CHECK(qFuzzyCompare(c.height(), 4.56));
 }
 
 TEST_CASE("Can copy a value QSizeF on the Rust side")
 {
   const auto s = QSizeF(1.23, 4.56);
   const auto c = copy_value_qsizef(s);
-  CHECK(c.width() == 1.23);
-  CHECK(c.height() == 4.56);
+  CHECK(qFuzzyCompare(c.width(), 1.23));
```

Comment:
I think the fuzzyCompare here is actually not necessary, as we're constructing & checking in C++, but it also shouldn't hurt.